src/test/java/com/stripe/StripeTest.java

```diff
@@ -7,50 +7,8 @@
 import com.stripe.exception.CardException;
 import com.stripe.exception.InvalidRequestException;
 import com.stripe.exception.StripeException;
-import com.stripe.model.Account;
-import com.stripe.model.Address;
-import com.stripe.model.ApplicationFee;
-import com.stripe.model.Balance;
-import com.stripe.model.BalanceTransaction;
-import com.stripe.model.BalanceTransactionCollection;
-import com.stripe.model.Card;
-import com.stripe.model.Charge;
-import com.stripe.model.ChargeCollection;
-import com.stripe.model.ChargeRefundCollection;
-import com.stripe.model.Coupon;
-import com.stripe.model.Customer;
-import com.stripe.model.CustomerSubscriptionCollection;
-import com.stripe.model.DeletedCard;
-import com.stripe.model.DeletedCoupon;
-import com.stripe.model.DeletedCustomer;
-import com.stripe.model.DeletedInvoiceItem;
-import com.stripe.model.DeletedPlan;
-import com.stripe.model.DeletedRecipient;
-import com.stripe.model.Dispute;
-import com.stripe.model.Event;
-import com.stripe.model.EvidenceDetails;
-import com.stripe.model.EvidenceSubObject;
-import com.stripe.model.FileUpload;
-import com.stripe.model.FraudDetails;
-import com.stripe.model.Invoice;
-import com.stripe.model.InvoiceItem;
-import com.stripe.model.InvoiceLineItemCollection;
-import com.stripe.model.MetadataStore;
-import com.stripe.model.Plan;
-import com.stripe.model.Recipient;
-import com.stripe.model.Refund;
-import com.stripe.model.Reversal;
-import com.stripe.model.ShippingDetails;
-import com.stripe.model.Subscription;
-import com.stripe.model.Token;
-import com.stripe.model.Transfer;
-import com.stripe.model.TransferReversalCollection;
+import com.stripe.model.*;
```

Comment:
I've historically advocated against wildcard imports (https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s3.3.1-wildcard-imports) but perhaps it's OK in tests?  OOC did your IDE do this automatically or such?